task_router_worker.go

```diff
@@ -0,0 +1,97 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+)
+
+const WorkersPathPart = "Workers"
+
+type WorkerService struct {
+	client       *Client
+	workspaceSid string
+}
+
+type Worker struct {
+	Sid          string `json:"sid"`
+	AccountSid   string `json:"account_sid"`
+	FriendlyName string `json:"friendly_name"`
+	// A string that contains JSON attributes, for example:
+	// `{"type": "support"}`
+	Attributes                    string     `json:"attributes"`
```

Comment:
As before the docs are incorrect, I've checked and `Attributes` returns a JSON string